src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -623,4 +623,7 @@
   <data name="Prefer_implicit_object_creation_when_type_is_apparent" xml:space="preserve">
     <value>Prefer implicit object creation when type is apparent</value>
   </data>
+  <data name="Insert_slash_slash_at_the_start_of_new_lines_when_writing_slash_slash_comments" xml:space="preserve">
+    <value>Insert // at the start of new lines when writing // comments</value>
```

Comment:
This is not much clear. The text is nearly identical to the other feature:

> Insert `*` at the start of new lines when writing `/* */` comments

This would make the users think that both features work the same. While they don't.

If I recall correctly, going to new line at the end of line with `/* */` comments adds asterisk.
However, going to new line at the end of line with `//` will not add `//`.

I believe it's better user experience if this is reworded to something different. Probably something like:

```suggestion
    <value>Insert // at the start of new lines when going to a new line in the middle of // comments</value>
```

Reply:
i don't really think that adds enough clarity to be warranted.

Reply:
I'm trying to find a better wording, but nothing comes to my head now.

However, I still think using the exact same wording when the behavior is different can make the users *initially* think that the feature will be annoying (while it's certainly not annoying)

Reply:
tbh, i don't expect users to ever interact with this or see this in general.  This is more an escape hatch for a user htat gets the new behavior and says "i don't like that".  we can point them here and they can just disable things.  I think all that's really necessary here is something they scan for to go "what is the thing that i can turn off for this new behavior in // comments".  And, in that regard, i think this string works fine :)

Reply:
I'm good with the current string here. Seems to fit well with the rest of the page.